tests/Test-DbaSpn.Tests.ps1

```diff
@@ -23,32 +23,18 @@ Describe $CommandName -Tag UnitTests {
 Describe $CommandName -Tag IntegrationTests {
     Context "When getting SPN information" {
         BeforeAll {
-            Mock Resolve-DbaNetworkName {
-                [PSCustomObject]@{
-                    InputName        = $env:COMPUTERNAME
-                    ComputerName     = $env:COMPUTERNAME
-                    IPAddress        = "127.0.0.1"
-                    DNSHostName      = $env:COMPUTERNAME
-                    DNSDomain        = $env:COMPUTERNAME
-                    Domain           = $env:COMPUTERNAME
-                    DNSHostEntry     = $env:COMPUTERNAME
-                    FQDN             = $env:COMPUTERNAME
-                    FullComputerName = $env:COMPUTERNAME
-                }
-            }
-            $results = Test-DbaSpn -ComputerName $env:COMPUTERNAME -WarningAction SilentlyContinue
+            $results = Test-DbaSpn -ComputerName $TestConfig.instance1
```

Comment:
🤔  how did this pass on appveyor?

Reply:
On AppVeyor this is the output:

<img width="712" height="385" alt="image" src="https://github.com/user-attachments/assets/1ff52106-5a07-4f72-89dc-ae6d1b382da4" />

Reply:
I think this test is only usefull in a bigger test environment with a domain and kerberos like I have it in my lab.